packages/management/tkg/bundle/config/values.yaml

```diff
@@ -13,3 +13,7 @@ frameworkPackage:
   featureGatesPackageValues:
     namespace: tkg-system
     createNamespace: false
+  tkrServicePackageValues:
+    namespace: tkg-system
```

Comment:
should this be tkr-system?

Reply:
No

Reply:
I think we shouldn't even give a choice of namespaces to sub-packages. 

The namespace we're installing this meta-package to should be provided to it, and *all* its sub packages should be installed into it. So we should probably just use the value of `namespaceForPackageInstallation` in the sub-packages instead of replicating it here multiple times.

Reply:
Since it affects all the meta-packages/sub-packages, we can discuss and have a followup PR.